File: MPI/mpicheck/tests/functions/2.0/MPI_File_get_view.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_File var_0;
    MPI_Offset *var_1;
    MPI_Datatype *var_2;
    MPI_Datatype *var_3;
    char *var_4;
    int ret;
    /* calls */
    ret = MPI_File_get_view(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_File_get_view(var_0, var_1, var_2, var_3, var_4);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_File_get_view.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER(KIND=MPI_OFFSET_KIND) var_1
       INTEGER var_2
       INTEGER var_3
       CHARACTER*(10) var_4
       INTEGER var_5
        call mpi_file_get_view(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_file_get_view(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    